lightning/src/util/config.rs

```diff
@@ -278,18 +293,8 @@ pub struct ChannelConfig {
 	pub cltv_expiry_delta: u16,
 	/// Deprecated, use [`ChannelHandshakeConfig::announced_channel`] instead.
 	pub(crate) announced_channel: bool,
-	/// When set, we commit to an upfront shutdown_pubkey at channel open. If our counterparty
-	/// supports it, they will then enforce the mutual-close output to us matches what we provided
-	/// at intialization, preventing us from closing to an alternate pubkey.
-	///
-	/// This is set to true by default to provide a slight increase in security, though ultimately
-	/// any attacker who is able to take control of a channel can just as easily send the funds via
-	/// lightning payments, so we never require that our counterparties support this option.
-	///
-	/// This cannot be changed after a channel has been initialized.
-	///
-	/// Default value: true.
-	pub commit_upfront_shutdown_pubkey: bool,
+	/// Deprecated, use [`ChannelHandshakeConfig::commit_upfront_shutdown_pubkey`] instead.
```

Comment:
Same here, you probably want to use:

```
#[deprecated(
	since = "0.0.107",
	note = "ChannelHandshakeConfig::commit_upfront_shutdown_pubkey should be used instead.",
)]
```